cmake/install.cmake

```diff
@@ -70,41 +75,36 @@ SET(INSTALL_DOCDIR_DEFAULT "docs")
 SET(INSTALL_LICENSEDIR_DEFAULT ${INSTALL_DOCDIR_DEFAULT})
 IF(NOT IS_SUBPROJECT)
   SET(INSTALL_PLUGINDIR_DEFAULT "lib/mariadb/plugin")
-ELSE()
 ENDIF()
 SET(LIBMARIADB_STATIC_DEFAULT "mariadbclient")
+
 #
 # RPM layout
 #
-SET(INSTALL_BINDIR_RPM "bin")
-IF((CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64" OR CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64" OR CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le" OR CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64" OR CMAKE_SYSTEM_PROCESSOR MATCHES "s390x") AND CMAKE_SIZEOF_VOID_P EQUAL 8)
-  SET(INSTALL_LIBDIR_RPM "lib64/mariadb")
-  SET(INSTALL_PCDIR_RPM "lib64/pkgconfig")
-  SET(INSTALL_PLUGINDIR_RPM "lib64/mariadb/plugin")
-ELSE()
-  SET(INSTALL_LIBDIR_RPM "lib/mariadb")
-  SET(INSTALL_PCDIR_RPM "lib/pkgconfig")
-  SET(INSTALL_PLUGINDIR_RPM "lib/mariadb/plugin")
-ENDIF()
-SET(INSTALL_INCLUDEDIR_RPM "include")
-SET(INSTALL_DOCDIR_RPM "docs")
+SET(INSTALL_BINDIR_RPM "${CMAKE_INSTALL_BINDIR}")
+SET(INSTALL_LIBDIR_RPM "${CMAKE_INSTALL_LIBDIR}/mariadb")
+SET(INSTALL_PCDIR_RPM "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
+SET(INSTALL_PLUGINDIR_RPM "${CMAKE_INSTALL_LIBDIR}/mariadb/plugin")
+SET(INSTALL_INCLUDEDIR_RPM "${CMAKE_INSTALL_INCLUDEDIR}/mariadb")
+SET(INSTALL_DOCDIR_RPM "${CMAKE_INSTALL_DOCDIR}")
+SET(INSTALL_LICENSEDIR_RPM "${INSTALL_DOCDIR_RPM}")
 SET(LIBMARIADB_STATIC_RPM "mariadbclient")
```

Comment:
Eh... `GNUInstallDirs` defines all those variables based on the current value of `CMAKE_INSTALL_PREFIX`, you have to include it **after** `CMAKE_INSTALL_PREFIX` is defined to its final value.

Reply:
ok thanks